### PR TITLE
COMMERCE-10604 PATCH a sku's ERC via API

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/COMMERCE-10604
Resent from https://github.com/liferay-commerce/liferay-portal/pull/3689

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSKUs.testcase [hunk 1]
```diff
@@ -254,6 +254,70 @@ definition {
...
+			CommerceProducts.searchAndGoToProductEntry(entryProductName = "MIN93015");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+			AssertTextEquals(
+				locator1 = "CommerceEntry#SKU_ERC_FIELD",
+				value1 = "MIN93015minium-initializer");
+		}
+
+		task ("When both null and empty values are patched and passed via API, the ERC for product is not updated") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSKUs.testcase:282]: After the "When" task, for the assertion, you can divide it into two tasks, first When task, second Then task for assertion. 

The same thing also in the And task that follows, first a When task, for assertion a Then task could be more clear.

**Comment by jaydawu** [comment]
ci:close

**Comment by jaydawu** [comment]
ci:reopen

**Comment by jaydawu** [comment]
ci:stop

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:stop

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:stop:relevant

**Comment by jaydawu** [comment]
ci:test:relevant

**Pull request closed without merging**